src/objectsAdvanced/task5.js

```diff
@@ -0,0 +1,16 @@
+const person = {
+    name: "Saule",
+    surname: "Gri",
+    age: "10",
+    birthDate: "2020",
+    fullName: function () {
+        this.fullName = this.name + ' ' + this.surname;
+    },
+    nickName: function () {
+        this.nickName = this.surname + ' ' + this.age;
+    }
+};
+
```

Comment:
Wow, impressing implementation! :) It would be great, that you understand all the details, how you accessed object fields from the function and so on. 
But the task was a bit different:
create a function.
it receives a person object
it returns a new object.
new object has two fields: fullName and nickname
these fields are constructed using person object fields.
Try to implement.